_rules/SC3-1-1-html-has-lang.md

```diff
@@ -99,12 +99,36 @@ The `lang` attribute specified has a non-empty value. The rule does not verify t
 
 #### Passed example 7
 
-The `xml:lang` attribute specified has a non-empty value. The rule does not verify the validity of the value specified and checks only for presence of a value.
+The `xml:lang` attribute specifies a value that is not empty (""). The rule does not verify the validity of the value specified and checks only for presence of a value.
 
 ```html
 <html xml:lang="xyz">
 ```
 
+#### Passed example 8
```

Comment:
I don't think we should put these new examples in. These are still failures of the SC.